Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -109,6 +116,10 @@ public ConceptEntry getWordnetConceptEntry(String wordnetId) throws LuceneExcept
 
     @Override
     public ConceptEntry getLocalConceptEntry(String conceptId) {
```

Comment:
it seems like this method is only used once (for deleting a concept after a merge). and only to avoid searching in merged ids when retrieving a concept. Why don't you just retrieve the concepts to be deleted before the merge using the getConceptEntry method and then delete those concepts after the merge has been successful?